openapi.yml

```diff
@@ -12988,6 +13030,93 @@ components:
         mitigation:
           type: string
         major_incident_state:
+          deprecated: true
+          oneOf:
+          - $ref: '#/components/schemas/MajorIncidentStateEnum'
+          - $ref: '#/components/schemas/BlankEnum'
+        major_incident_start_dt:
+          type: string
+          format: date-time
+          nullable: true
+        nist_cvss_validation:
+          oneOf:
+          - $ref: '#/components/schemas/NistCvssValidationEnum'
+          - $ref: '#/components/schemas/BlankEnum'
+        aegis_meta: {}
+        embargoed:
+          type: boolean
+          description: The embargoed boolean attribute is technically read-only as
+            it just indirectly modifies the ACLs but is mandatory as it controls the
+            access to the resource.
+        group_key:
+          type: string
+          maxLength: 60
+        owner:
+          type: string
+          maxLength: 60
+        team_id:
+          type: string
+          maxLength: 8
+      required:
+      - comment_zero
+      - embargoed
+      - title
```

Comment:
I wonder where this change came from? 🤔 Only thing which changed for `FlawPostSerializer` was `deprecate_fields=["major_incident_state"]`